src/tools/rbd_mirror/image_replayer/ReplayStatusFormatter.cc

```diff
@@ -201,16 +202,16 @@ void ReplayStatusFormatter<I>::handle_update_tag_cache(uint64_t master_tag_tid,
       tag_data.predecessor.mirror_uuid !=
         librbd::Journal<>::ORPHAN_MIRROR_UUID) {
     dout(20) << "hit remote image non-primary epoch" << dendl;
-    tag_data.predecessor.tag_tid = mirror_tag_tid;
+    tag_data.predecessor = {"", true, mirror_tag_tid, 0};
   } else if (tag_data.predecessor.tag_tid == 0) {
```

Comment:
Nit: this conditional can now be dropped since the tag is already zeroed if the retrieve/decode failed